### PR TITLE
Feature/48 es lint doc

### PR DESCRIPTION
**Pull request by kevinmpowell**
References: https://github.com/EightShapes/esds-project/issues/48

Same structure as sass linting doc.

### CODE REVIEW

File: components/README.md [hunk 1]
```diff
@@ -3,19 +3,36 @@
...
 
 The ruleset for each component is managed in `components/**/.stylelintrc.json`. Component level rulesets enable future coding style changes to be applied on a component-by-component basis.
 
 Stylelint is paired with [prettier](https://prettier.io) to automatically correct code style. In supported text editors this can be performed each time a file is edited.
 
 ##### Pre-commit hook
-Stylelint and prettier are called automatically on each `git commit` using [husky](https://github.com/typicode/husky#readme). The pre-commit hook will attempt to correct any style discrepancies automatically. If it cannot, an error will be shown at the command line describing any linting failures and the commit will be aborted.
+Stylelint and prettier are called automatically on each `git commit` using [husky](https://github.com/typicode/husky#readme). The pre-commit hook will attempt to correct any lint discrepancies automatically. If it cannot, an error will be shown at the command line describing any linting failures and the commit will be aborted.
```
Comment (nathanacurtis) [inline components/README.md]: I'd be curious how such errors are shown in a Git UI like Tower.

Comment (kevinmpowell) [reply components/README.md]: Good news Tower users, tower surfaces git hook console messages:
image

Comment (nathanacurtis) [reply components/README.md]: Curious: does this happen upon submitting from the Commit dialog (with title and description), or upon push from local to remote?

Comment (kevinmpowell) [reply components/README.md]: Upon submitting from Commit. It's a pre-commit hook vs. a pre-push hook. Prevents you from polluting your local git history with unlinted code. In many git workflows multiple commits are made locally (sometimes over long periods of time) before a push to the remote. I typically: commit locally, immediately push, but that's not always common.

**Pull request merged**